code/modules/antagonists/clock_cult/mobs/eminence.dm

```diff
@@ -293,8 +290,6 @@
 		"Processor Overload",
 		"Radiation Storm"
 	)
-	if(!can_cast(user))
-		return
```

Comment:
Removing this might be a bad idea, maybe `cast_check(TRUE, user)` could work here?

Reply:
I'm pretty sure that can_cast is run elsewhere earlier in the call chain of the button.

Reply:
Yea but it needs to be here because it is after a user input which is blocking

Reply:
Hello?